nautobot_floor_plan/models.py

```diff
@@ -79,6 +81,37 @@ def get_svg(self, *, user, base_url):
         """Get SVG representation of this FloorPlan."""
         return FloorPlanSVG(floor_plan=self, user=user, base_url=base_url).render()
 
+    def save(self, *args, **kwargs):
+        """Override save in order to update any existing tiles."""
+        if not self.created:
+            super().save(*args, **kwargs)
+            return
+        # Get origin_seed pre/post values
+        initial_instance = self.__class__.objects.get(pk=self.pk)
+        x_initial = initial_instance.x_origin_seed
+        y_initial = initial_instance.y_origin_seed
+        x_updated = self.x_origin_seed
+        y_updated = self.y_origin_seed
+
+        super().save(**kwargs)
+
+        if x_initial != x_updated or y_initial != y_updated:
+            self.update_tile_origins(x_initial, x_updated, y_initial, y_updated)
+
+    def update_tile_origins(self, x_initial, x_updated, y_initial, y_updated):
+        """Update any existing tiles if axis_origin_seed was modified."""
+        tiles = self.tiles.all()
+        x_delta = x_updated - x_initial
+        y_delta = y_updated - y_initial
+
+        # should bulk_update these instead?
+        with transaction.atomic():
```

Comment:
Wondering if this atomic transaction should instead be in the `save()` method and wrap both the `super().save()` and the call to this method?

Reply:
Agreed, will do.